test/framework/easyconfigs/test_ecs/s/ScaLAPACK/ScaLAPACK-2.0.2-gompi-1.4.10-OpenBLAS-0.2.6-LAPACK-3.4.2.eb

```diff
@@ -20,7 +20,7 @@ blassuff = '-LAPACK-3.4.2'
 
 versionsuffix = "-%s-%s%s" % (blaslib, blasver, blassuff)
 
-dependencies = [(blaslib, blasver, blassuff)]
+dependencies = [(blaslib, blasver, blassuff, ('GCC', '4.7.2'))]
```

Comment:
@bartoldeman There should be no need to hardcode to `GCC` here, see most recent `ScaLAPACK` easyconfigs like `ScaLAPACK-2.0.2-gompic-2018b-OpenBLAS-0.3.1.eb`.

Reply:
did my other comments get lost?

Reply:
What other comments? Maybe that answers your question. ;)